### DISCUSSION TITLE
Destek Tagı--- HERKES HERKESİN OKUNU TIKLASIN BEN BAŞLIYORUM.

### DISCUSSION THREAD

**Discussion by zukenberger** [IBAX-io/go-ibax#956] [Soru Cevap - Türkçe]
Arkadaşlar birbirimize destek olalım. Herkes arkadaşların yazdıklarına yorum yazsın ve yukarıya dönük mavi oka bassın. ben başlıyorum. Buna yorum yapan herkesin okuna basacağım. Herkes biribirine bassın....

**Comment by Yusufknk** [comment]
🎉
